docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_BERT_From_Tensorflow.md

```diff
@@ -110,10 +109,9 @@ python3 run_classifier.py \
 
 Run the Model Optimizer with the following command line parameters to generate reshape-able BERT Intermediate Representation (IR):
 ```sh
-python3 ./mo_tf.py
---input_model inference_graph.pb 
---input "IteratorGetNext:0{i32}[1 128],IteratorGetNext:1{i32}[1 128],IteratorGetNext:4{i32}[1 128]" 
---disable_nhwc_to_nchw 
+python3 ./mo_tf.py \
+    --input_model inference_graph.pb  \
+    --input "IteratorGetNext:0{i32}[1 128],IteratorGetNext:1{i32}[1 128],IteratorGetNext:4{i32}[1 128]" 
```

Comment:
@pavel-esir, please pay attention to feature you recently implemented for `IteratorGetNext`? Can we convert this particular model OOB?

@mvafin, it does not relate to your PR. please ignore :)